### PR TITLE
Updating GO_VERSION -> 1.19, removing depreacted ioutil import

### PR DESCRIPTION
**Pull request by moadz**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,14 +1,12 @@
 module github.com/observatorium/opa-ams
 
-go 1.14
+go 1.19
```
Comment (squat) [inline go.mod]: Is there any reason we need to bump this? Do any of our deps use generics or new features?

Comment (moadz) [reply go.mod]: Not directly, was touching the repo so figured I might as well. If we don't want to bump versions, happy to close the PR.

Comment (squat) [reply go.mod]: Bumping versions is cool! But this particular line indicates the minimum version needed by this module, not what the binary is built with

Comment (moadz) [reply go.mod]: You're correct, it was more to avoid confusion around the deprecated `ioutil` packages if compiling with a >1.16 version. 

Will fix.

Comment (squat) [reply go.mod]: Ah I see 🌟

**Review by saswatamcode — APPROVED** [review]
LGTM!

**Pull request merged**